code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/gun/energy/taser
 	name = "taser gun"
 	desc = "A small, low capacity gun used for non-lethal takedowns."
+	weapon_weight = WEAPON_HEAVY
```

Comment:
Giving heavy weight to the taser gun?

Reply:
Not intentional, just a mistake I didn't notice. Correcting it in next commit.